cl/lib/decorators.py

```diff
@@ -46,7 +46,7 @@ def f_retry(*args, **kwargs):
                 try:
                     return f(*args, **kwargs)
                 except ExceptionToCheck as e:
-                    msg = "%s, Retrying in %d seconds..." % (str(e), mdelay)
+                    msg = "{e}, Retrying in {mdelay} seconds..."
```

Comment:
This line is literally rendering  `{e}, Retrying in {mdelay} seconds...`  every time

Reply:
Woops! This one could maybe have been detected by this in-preview rule: https://docs.astral.sh/ruff/rules/missing-f-string-syntax/

I think best to avoid preview rules if we can though...